test/transforms/fixtures/at-current-range/delete-backward/inside-inline-with-trailing-newline/index.js

```diff
@@ -0,0 +1,28 @@
+
+import assert from 'assert'
+
+export default function (state) {
+  const { document, selection } = state
+  const texts = document.getTexts()
+  const first = texts.get(1)
+  const range = selection.merge({
+    anchorKey: first.key,
+    anchorOffset: 0,
+    focusKey: first.key,
+    focusOffset: 0
+  })
+
+  const next = state
+    .transform()
+    .moveTo(range)
+    .deleteBackward()
+    .apply()
+
+  // Not sure what this should look like
+  // assert.deepEqual(
+  //   next.selection.toJS(),
+  //   range.toJS()
```

Comment:
Either the selection has a bug here too, or I made a mistake. Please let me know what should be here